netbout-web/src/main/java/com/netbout/rest/account/TkSaveEmail.java

```diff
@@ -120,14 +122,8 @@ public Response act(final Request req) throws IOException {
                 "%semverify/%s",
                 new RqHref.Smart(new RqHref.Base(req)).home().bare(), code
             );
-            final String old;
-            if (alias.email().contains("!")) {
-                old = alias.email().substring(0, alias.email().indexOf('!'));
-            } else {
-                old = alias.email();
-            }
             try {
-                alias.email(String.format("%s!%s", old, email), link);
+                alias.email(combinedEmails(alias, email), link);
```

Comment:
@CarlosGines Same here. Please address it in a static way `TkSaveEmail.combinedEmails()`